raiden/transfer/mediated_transfer/mediator.py

```diff
@@ -242,7 +242,7 @@ def get_lock_amount_after_fees(
     # fee_out should be calculated on the payment amount without any fees. But
     # we only have the amount including fee_out, so we use that as an
     # approximation.
-    fee_out = _fee_for_channel(payee_channel, PaymentAmount(lock.amount - fee_in))
```

Comment:
@karlb Please look at this change in detail.

Reply:
Pro:
* Make the rest of this PR easier
* Makes it easier to calculate Fees in your head with the usual settings (same fees on all channels)
Cons:
* The mapping between proportional fee and locked tokens is less accurate
* Will probably require changes in the PFS and its tests so that fee estimate and the fees match up properly. (btw. we could use tests for that, too!)

I'm a bit torn on this. Let me sleep over it.

Reply:
> Let me sleep over it.

I like that. Let's discuss it tomorrow.

Reply:
I'm still not completely sure, but I think I prefer keeping it the old way. That makes it easier to think about both channels separately and feels more straightforward when working with the general case (proportional fees differ between channels). Using a small formula when processing the user input does not seem too bad.